nt-be/src/lib.rs

```diff
@@ -0,0 +1,68 @@
+pub mod constants;
+pub mod handlers;
+pub mod routes;
+pub mod utils;
+
+use moka::future::Cache;
+use near_api::{NetworkConfig, RPCEndpoint};
+use sqlx::PgPool;
+use std::time::Duration;
+
+pub struct AppState {
+    pub http_client: reqwest::Client,
+    pub cache: Cache<String, serde_json::Value>,
+    pub network: NetworkConfig,
+    pub archival_network: NetworkConfig,
+    pub env_vars: utils::env::EnvVars,
+    pub db_pool: PgPool,
+}
+
+/// Initialize the application state with database connection and migrations
+pub async fn init_app_state() -> Result<AppState, Box<dyn std::error::Error>> {
+    // Database connection
+    let database_url = std::env::var("DATABASE_URL")?;
```

Comment:
Add to env_vars to keepp all of them in same place

Reply:
thanks, will fix that in the next PR